guides/common/modules/ref_best-practices-for-errata.adoc

```diff
@@ -0,0 +1,24 @@
+[id="best-practices-for-errata_{context}"]
+= Best practices for errata
+
+* {Project} shows which errata are applicable for each content host by comparing package lists with installed and available packages.
+* Use errata to add patches for security issues to a frozen set of content without unnecessarily updating other unaffected packages.
```

Comment:
* Fetch information about host errata by using the Foreman link:https://theforeman.github.io/foreman-ansible-modules/develop/plugins/host_errata_info_module.html#ansible-collections-theforeman-foreman-host-errata-info-module[host_errata_info module] in an Ansible playbook.